js/ui/map.js

```diff
@@ -586,7 +594,7 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
         } else if (style instanceof Style) {
             this.style = style;
         } else {
-            this.style = new Style(style, this.animationLoop);
+            this.style = new Style(style, this.animationLoop, this._workerCount);
```

Comment:
I have a preference towards passing `options.workerCount` directly to `Style`. I'm trying to push `Map` in the direction of becoming a dumb [facade](https://en.wikipedia.org/wiki/Facade_pattern) class because it is currently a [god object](https://en.wikipedia.org/wiki/God_object).